### ISSUE TITLE
Minor issues with compact density (W10 - 76)

### ISSUE THREAD

**Issue by s-rog** [coekuss/quietfox#9]
![Screenshot (7)](https://user-images.githubusercontent.com/55400948/81288065-7c1d2380-9096-11ea-9504-f8a1de062281.png)

Minor issues:
- no padding left of the back button
- no padding on top of tabs
- url bar cuts off some letters (g)

In this screenshot I also removed the border around the url bar which isn't present in your screenshots on the readme, not sure if it was supposed to be there... theme works and looks great otherwise, cheers!

**Comment by coekuss** [comment]
You might be using an older version of Quietfox, since the border around the url bar is something I have recently fixed, unless you're talking about something else. However I do agree the left back button should have padding so I just [released a fix for that](https://github.com/coekuss/quietfox/releases/tag/v6)

**Comment by s-rog** [comment]
![Screenshot (12)](https://user-images.githubusercontent.com/55400948/81358164-53ce0d00-9108-11ea-9661-93167fa16fe0.png)
the button padding is fixed! but as you can see there is the border around the bar... also even under normal density there is no padding on top of tabs (unlike your screenshots) but the url bar only cuts off letters in compact

**Comment by coekuss** [comment]
Forgot to mention the padding on the top of tabs is an option in the Firefox customization

![image](https://user-images.githubusercontent.com/45906331/81359029-08831280-90a6-11ea-87d0-1acf02905374.png)

Also strange that I'm not seeing the urlbar border or the letters being cut off on my end. Line 520 makes the urlbar border transparent: https://github.com/coekuss/quietfox/blob/master/chrome/userChrome.css#L520. However, I did just add your suggestion to the code. I also removed vertical padding inside the urlbar so it should no longer be cut off.
